doc/WebOnChain.plantuml

```diff
@@ -0,0 +1,186 @@
+@startuml
+participant "End user" as user
+participant "Browser" as ui
+participant "Thyra server" as thyra
+participant "Hosting websites" as web
+participant "Wallet" as w
+participant "DNS" as dns
+participant "Event Manager" as em
+participant "Storage" as storage
+participant "Node (API)" as node
+
+user -> ui: click on install 'Web on-chain' plug-in
```

Comment:
You should add a [divisor](https://plantuml.com/fr/sequence-diagram#d4b2df53a72661cc) here to make thing clearer.

Furthermore you should move this part to a dedicated diagram. This schema is to big.
```suggestion
== Plug-in installation ==

user -> ui: click on install 'Web on-chain' plug-in
```

Reply:
ack, will share an updated and lighter version.